File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/repository/jpa/CakeShopLikeJpaRepository.java
package com.cakk.domain.mysql.repository.jpa;

import org.springframework.data.jpa.repository.JpaRepository;

import com.cakk.domain.mysql.entity.shop.CakeShopLike;
import com.cakk.domain.mysql.entity.user.User;

public interface CakeShopLikeJpaRepository extends JpaRepository<CakeShopLike, Long> {

	int countByCakeShopIdAndUser(Long cakeShopId, User user);
}

File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/repository/jpa/CakeShopLikeJpaRepository.kt
package com.cakk.domain.mysql.repository.jpa

import com.cakk.domain.mysql.entity.shop.CakeShopLike
import com.cakk.domain.mysql.entity.user.User
import org.springframework.data.jpa.repository.JpaRepository

interface CakeShopLikeJpaRepository : JpaRepository<CakeShopLike, Long> {
    fun countByCakeShopIdAndUser(cakeShopId: Long, user: User): Int
}
